### ISSUE TITLE
Class 'verbb\formie\fields\formfields\Json' not found

### ISSUE THREAD

**Issue by TomDeSmet** [verbb/formie#283]
**Description**
Just updated to 1.3.7 and getting this error.
![Screen Shot 2021-01-08 at 09 35 09 AM](https://user-images.githubusercontent.com/16959207/103993146-6fa51980-5195-11eb-987f-e994cb05874d.png)

The cullprit is an HTML field I'm using. If I remove that field, it works.
Probably related to an update in 1.3.5: `Added ability to provide htmlpurifier config JSON files for HTML fields.`

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.8](https://github.com/verbb/formie/releases/tag/1.3.8).
